CMakeLists.txt

```diff
@@ -421,7 +423,7 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
 	# the script we created above is now added as a command to run at
 	# install time - so this ensures that subsurface.exe has been
 	# built before this is run
-	install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER} -DSUBSURFACE_TARGET=${SUBSURFACE_TARGET} -DSUBSURFACE_SOURCE=${CMAKE_SOURCE_DIR} -DSTAGING=${WINDOWSSTAGING} -P ${CMAKE_SOURCE_DIR}/cmake/Modules/dlllist.cmake)")
+	install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER} -DSUBSURFACE_TARGET=${SUBSURFACE_TARGET} -DSUBSURFACE_SOURCE=${SUBSURFACE_SOURCE} -DSTAGING=${WINDOWSSTAGING} -P ${CMAKE_SOURCE_DIR}/cmake/Modules/dlllist.cmake)")
```

Comment:
I'm not clear how this works. You set SUBSURFACE_SOURCE to be CMAKE_SOURCE_DIR
How is this different from just using CMAKE_SOURCE_DIR.
I must be missing something